modules/ROOT/pages/roles.adoc

```diff
@@ -3,9 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To manage roles and permissions within Anypoint Platform, you must have the Organization Administrators role. To manage user permissions for an API version, you must have the API Versions Owner role.
+A role is a set of pre-defined permissions for each product, or feature, within Anypoint Platform. Depending on the product, you can use pre-defined roles with standard permissions, or you can specify your own permissions for each role. Roles can be set only for individual users. To assign the same permissions to multiple users at once, use the xref:access-management::teams.adoc[Teams feature].
```

Comment:
Roles apply to multiple users. The benefits that teams provide are:
- They work x-business group
- They have inheritance semantics